.github/workflows/ci.yml

```diff
@@ -0,0 +1,42 @@
+name: CI
+
+on:
+  pull_request:
+    branches: [main]
+
+  push:
+    branches: [assignment_2.1]

```

Comment:
Use the `push` trigger only for your own testing purpose.
But in the final submission have only the `pull_request`.
Since you had both the triggers, the same pipeline ran twice..

![image](https://github.com/saachi-tiger/mle-repository/assets/53215532/2840783e-0e1e-4873-b604-32ce577b868f)

So have only one trigger (`pull_request`) in the final submission.

Reply:
Remove push on branch. Build is only created for PR to main